extensions/ql-vscode/src/data-extensions-editor/data-extensions-editor-module.ts

```diff
@@ -99,13 +110,68 @@ export class DataExtensionsEditorModule {
               return;
             }
 
+            // Create new temporary directory for query files and pack dependencies
```

Comment:
Optional suggestion: could this chunk of code being moved here go into a separate method? It would help to keep the implementation of the `codeQL.openDataExtensionsEditor` command easy to read.

Would it actually make sense to keep this code in `external-api-usage-query.ts`, so then it's co-located with the `runQuery` method which depends on the setup being done by this code.

Reply:
Good idea! That also simplifies tests I think.

Reply:
I made some changes